src/OpenLoco/Windows/Map.cpp

```diff
@@ -1090,7 +1090,7 @@ namespace OpenLoco::Ui::Windows::Map
 
         xy32 startPos = { Location::null, 0 };
         xy32 endPos = { Location::null, 0 };
-        for (auto& order : Vehicles::OrderTableView(train.head->orderTableOffset))
+        for (auto& order : Vehicles::OrderRingView(train.head->orderTableOffset, 0))
```

Comment:
Might be nicer to have `0` be the default value for the second parameter. If it isn't, it's probably in a variable of some sort, anyway. Right now, it's not clear that this `0` refers to the initial offset.